.github/workflows/e2e_email_dashboard_features.yml

```diff
@@ -0,0 +1,47 @@
+name: End-to-End tests
+on:
+  push:
+    branches:
+      - develop
+      - release-*
+  pull_request:
+    branches:
+      - develop
+      - release-*
+
+jobs:
+  e2e_email_dashboard_features:
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '2.x'
+          architecture: 'x64'
+      - name: Setup python by installing wheel
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: pip install wheel==0.35.0
+      - name: Install third party
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: python -m scripts.install_third_party_libs
+      - name: Install chrome version 88.0.4324.96-1
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: |
+          sudo apt-get update
+          sudo apt-get install libappindicator3-1
+          curl -L -o google-chrome.deb https://github.com/webnicer/chrome-downloads/raw/master/x64.deb/google-chrome-stable_88.0.4324.96-1_amd64.deb
+          sudo sed -i 's|HERE/chrome\"|HERE/chrome\" --disable-setuid-sandbox|g' /opt/google/chrome/google-chrome
+          sudo dpkg -i google-chrome.deb
+          google-chrome --version
+      - name: Run Email Dashboard E2E Test
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: xvfb-run -a python -m scripts.run_e2e_tests --suite="emailDashboard" --prod_env
```

Comment:
Can we move this e2e test to any existing e2e test group? (maybe the group that test dashboard)

@U8NWXD, Do you have any suggestion?

Reply:
I think it's fine to keep separate. We're actually soon going to be splitting up the workflows more to take advantage of parallelization